### DISCUSSION TITLE
Command-line tool with Python integration

### DISCUSSION THREAD

**Discussion by Fidolorous** [LibreScore/dl-librescore#71] [Q&A] [answered]
Hello all, thank you for this wonderful tool.

I would like to connect the Command-line tool with a Python script which would let you log into your premium account and then let you download .pdfs in bulk, instead of manually searching for each link.

Do you think that would be possible to implement?

**Comment by PeterNjeim** [comment, marked as answer]
The command-line tool is written in Javascript, so Python integration isn't possible. Instead you can make a Python script that runs terminal commands just like you would normally. For bulk downloads, you can create a text file with the URL's you want to download, then make the script take that as an input and parse it line by line. As for sharing your account credentials, unfortunately that's not on our to-do list but we'll consider it
